mimic/imimic.py

```diff
@@ -28,13 +28,37 @@ def resource_for_region(region, uri_prefix,
         """
 
 
+class IEndPointTemplate(Interface):
+    """
+    AN :obj:`IEndPointTemplate` provides an interface for the OS-KSCATALOG
+    Admin end-point templates, defining the fields.
```

Comment:
It looks like there is a mixture of `end-point` and `endpoint`. Would you mind using `endpoint` in the docstrings? It looks to be the chosen spelling convention in mimic to date.